tests/nn/conv/test_simple_conv.py

```diff
@@ -0,0 +1,66 @@
+import mlx.core as mx
+
+from mlx_graphs.nn.conv.simple_conv import SimpleConv
+
+mx.random.seed(42)
```

Comment:
is this needed?

Reply:
Good point. This is not needed. It will be removed.